nilearn/plotting/surf_plotting.py

```diff
@@ -369,30 +380,35 @@ def _get_cmap_matplotlib(cmap, vmin, vmax, threshold=None):
 
 
 def _compute_facecolors_matplotlib(bg_map, faces, n_vertices,
-                                   darkness, alpha):
+                                   darkness, alpha, bg_map_rescale):
     """Helper function for plot_surf with matplotlib engine.
 
     This function computes the facecolors.
     """
     # set alpha if in auto mode
     if alpha == 'auto':
         alpha = .5 if bg_map is None else 1
+
     if bg_map is None:
         bg_data = np.ones(n_vertices) * 0.5
     else:
         bg_data = load_surf_data(bg_map)
         if bg_data.shape[0] != n_vertices:
             raise ValueError('The bg_map does not have the same number '
                              'of vertices as the mesh.')
+
     bg_faces = np.mean(bg_data[faces], axis=1)
-    if bg_faces.min() != bg_faces.max():
+    if bg_map_rescale and bg_faces.min() != bg_faces.max():
```

Comment:
here if `bg_map_rescale` is `"auto"` this becomes `True` -- `bg_map_rescale="auto"` is treated the same as `bg_map_rescale=True`, bg maps with values in ]0, 1[ still get normalized

Reply:
Yes, good catch, I had forgotten to update this function...
I just updated it so that it matches now `_get_vertexcolor()` and added tests so that we control its behaviour for different values of `bg_map_rescale`.